### DISCUSSION TITLE
Larastan not finding methods on models

### DISCUSSION THREAD

**Discussion by edalzell** [larastan/larastan#1639] [Q&A]
On level 8, I'm getting errors like this:

<img width="685" alt="CleanShot 2023-05-06 at 21 15 01@2x" src="https://user-images.githubusercontent.com/6069653/236657471-877cb483-5ad1-45a5-a94f-0a7f863ad7dc.png">

And it's defined here:

<img width="911" alt="CleanShot 2023-05-06 at 21 22 59@2x" src="https://user-images.githubusercontent.com/6069653/236657489-e8f9e61e-cb14-4200-92a5-045263893fc9.png">

Why would this be an error?

**Comment by edalzell** [comment]
It looks like it's the IDE Helper getting in the way here, interesting.

**Reply by szepeviktor** [reply]
Yes. Please consider replacing ide-helper with an IDE add-on/extension for Laravel, preferably a paid one.
